tests/test_common/test_utils/test_checkpoints.py

```diff
@@ -0,0 +1,37 @@
+import os
+import unittest
+
+import numpy as np
+import torch
+import torch.optim as optim
+import torchvision.models as models
+
+from code_soup.common.utils.checkpoints import Checkpoints
```

Comment:
This line should be from code_soup.common.utils import Checkpoints
and
from code_soup.common.utils.checkpoints import Checkpoints should be there in code_soup/common/utils/__init__.py